src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -236,11 +238,16 @@ namespace HVACVariableRefrigerantFlow {
     Array1D<VRFTUNumericFieldData> VRFTUNumericFields; // holds VRF TU numeric input fields character field name
 
     void SimulateVRF(std::string const &CompName,
-                     int const ZoneNum,
```

Comment:
Sim function arguments reorganized so that calls as zone/airloop/OA system are similar.